### PR TITLE
[release-4.9] [windows] Increase kubelet verbosity in debug

### PR DESCRIPTION
**Pull request by jrvaldes**
This is manual cherry-pick of https://github.com/openshift/windows-machine-config-operator/pull/1198

### CODE REVIEW

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

File: pkg/windows/windows.go [hunk 1]
```diff
@@ -562,6 +562,11 @@ func (vm *windows) runBootstrapper() error {
 	if vm.clusterDNS != "" {
 		wmcbInitializeCmd += " --cluster-dns " + vm.clusterDNS
 	}
+	// check log level and increase kubelet verbosity accordingly
```
Comment (aravindhp) [inline pkg/windows/windows.go:565]: Please add a new line before the comment to be consistent with the code in master

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Pull request merged**